.template/addons/bootstrap/package.json.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+insert_into_file 'package.json', after: /"i18n-js":.+\n/ do
```

Comment:
How about add this after `"dependencies"`? If `i18n-js` get removed in any case I'm not sure if insertion will still work.

Reply:
You are right. But to make sure that insertion worked we write tests for it. As we have insertions like this all over the template 😅 I feel like we should change everywhere 😅 Do you think it's necessary? 🤔

Reply:
My bad, I forget about the tests. We can go with your solution as we can make sure that this will be added according to test.